sim/mcuboot-sys/csupport/run.c

```diff
@@ -449,3 +449,13 @@ void sim_assert(int x, const char *assertion, const char *file, unsigned int lin
         }
     }
 }
+
+uint32_t get_boot_max_align(void)
```

Comment:
For our typical nomenclature, it would be better if those functions didn't have the `get_` prefix.